scripts/check-release-please.mjs

```diff
@@ -0,0 +1,82 @@
+import { strict as assert } from 'assert';
+import * as childProcess from 'child_process';
+import path from 'path';
+import { readFileSync } from 'fs';
+
+/*
+	Enumerates through
+
+	- lerna packages,
+	- release please's manifest, and
+	- release please's config, and
+
+	makes sure all those lists are in sync. If not, guides you to do the correct thing
+*/
+
+const errors = [];
+const logErrorIf = (condition, errorMessage) => {
+	if (condition) {
+		errors.push(errorMessage);
+	}
+};
+const PROJECT_ROOT = path.resolve();
+const readJson = (path) => {
+	return JSON.parse(readFileSync(path));
+};
+const getProcessOutput = (cmd, args) => {
+	const result = childProcess.spawnSync(cmd, args);
+	assert(!result.error, result.error);
+	return result.stdout.toString('utf8');
+}
+
+const lernaList = JSON.parse(
+getProcessOutput('lerna', ['list', '--json'])
+);
+const packageList = new Map(
+	lernaList.map((pkg) => {
+		const relativeLocation = path.relative(PROJECT_ROOT, pkg.location);
+		return [relativeLocation, { ...pkg, relativeLocation: path.relative(PROJECT_ROOT, pkg.location) }];
+	})
+);
+const manifest = readJson('.release-please-manifest.json');
+const config = readJson('release-please-config.json');
+
+const manifestPackages = new Set(Object.keys(manifest));
+const configPackages = new Set(Object.keys(config.packages));
+
+console.log('lerna packages', );
+console.log('manifest packages', manifestPackages);
+console.log('config packages', configPackages);
+
+packageList.forEach(({ relativeLocation }) => {
+	logErrorIf(
+		!manifestPackages.has(relativeLocation),
+		`Could not find ${relativeLocation} from manifest. If you are adding a new package. Add following to .release-please-manifest.json
+  "${relativeLocation}": "0.0.1",`);
```

Comment:
I thought we start usually with `0.1.0`?

Reply:
The first release is then `0.2.0`. See [dataloader's first release](https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/instrumentation-dataloader-v0.2.0).

Haven't validated, but if it bumps minor then `0.0.1` -> `0.1.0`.